pymc/distributions/continuous.py

```diff
@@ -594,7 +597,7 @@ class TruncatedNormalRV(RandomVariable):
     def rng_fn(
         cls,
         rng: np.random.RandomState,
-        mu: Union[np.ndarray, float],
+        mu: DIST_PARAMETER_TYPES,
```

Comment:
In the `rng_fn` the previous type hints were correct, it's in the `.dist` classmethod in the distribution below that the `TensorVariable` also shows up as a possibility